packages/web/src/features/translation/TranslateWord.tsx

```diff
@@ -107,6 +112,14 @@ const TranslateWord = forwardRef<TranslateWordRef, TranslateWordProps>(
           <span className="inline-block" ref={ancientWord}>
             {word.text}
           </span>
+          <Button
+            className={hasNote ? 'inline-block' : 'hidden'}
+            small
+            variant="tertiary"
+            onClick={onOpenNotes}
```

Comment:
We are going to want a `tabIndex` of -1 so that this isn't in the tab sequence. That will be tedious to navigate through each note icon.

Reply:
Done.